src/ezsocket/socketWorker.cxx

```diff
@@ -108,8 +106,8 @@ void SocketWorker::worker()
     uint32_t readyBytes;
     int32_t recvSize;
     int32_t sendSize;
-    while (true)
-    {
+    while (true){
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
@EulersBoiler Good idea. But I think we don't use the SocketWorker anywhere anymore, not sure though.